portal/src/components/DoDontExample/DoDontExample.tsx

```diff
@@ -21,11 +22,16 @@ export function DoDontExample({ type, image, description, fullWidth }: Props) {
     });
     const containerClass = classNames("jkl-portal-do-dont-example", {
         "jkl-portal-do-dont-example--fullwidth": fullWidth,
+        "jkl-portal-do-dont-example--halfwidth": !fullWidth,
     });
 
     return (
         <aside className={containerClass}>
-            <img className="jkl-portal-do-dont-example__image" src={withPrefix(image)} alt={altText} />
+            {content ? (
+                <div>{content}</div>
```

Comment:
```suggestion
                <div className="jkl-portal-do-dont-example__image">{content}</div>
```
så slipper du vel `& img`-selektoren i stilarket?

Reply:
content is content, you need to pass classes manually